source/fundamentals/connection/network-compression.txt

```diff
@@ -16,21 +16,19 @@ The driver supports the following compression algorithms:
 If you specify multiple compression algorithms, the driver selects the
 first one in the list supported by your MongoDB instance.
 
-.. note::
-
-   When using the Snappy or Zstandard compression algorithm, you must
-   :ref:`add explicit dependencies <compression-dependencies>`.
-
 .. _enable-compression:
 
 Specify Compression Algorithms
 ------------------------------
 
 You can enable compression for the connection to your MongoDB instance
-by specifying the algorithms in one of two ways: 
+by specifying the algorithms using one of the following methods:
```

Comment:
I: [Avoid "using" on its own](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#u)

S: 

```suggestion
by using one of the following methods to specify the algorithm:
```
Or reword the whole sentence:

```suggestion
To enable compression for the connection to your MongoDB instance,
use one of the following methods to specify the algorithm:
```